src/lib/components/search/load/Loads.svelte

```diff
@@ -12,31 +14,45 @@
 
 	// parameters
 	export let query;
+	export let hidden = true;
+	export let delay = 100; // test value and change accordingly
 
 	// variables
 	const category = categoryFrom(query);
 	const numberOfResults = 32;
+
+	onMount(() => {
+		setTimeout(() => {
+			hidden = false;
+		}, delay);
+	});
 </script>
 
-{#if category === CategoryEnum.IMAGES}
-	<div class="px-4 py-8">
-		<section id="images" class="flex flex-wrap justify-center gap-2">
-			{#each { length: numberOfResults } as _}
-				<div class="flex-none">
-					<LoadImage />
-				</div>
-			{/each}
-		</section>
-	</div>
-{:else if category !== undefined}
-	<div class="sm:mx-auto mb-4 max-w-screen-sm">
-		<section id="result-list" class="mx-2 my-4 max-w-fit overflow-clip">
-			{#each { length: numberOfResults } as _}
-				<Load />
-				<hr class="my-2 border border-gray-200 dark:border-gray-600" />
-			{/each}
-		</section>
-	</div>
-{:else}
-	<Error statusCode={'500'} message={'Unknown category requested.'} />
-{/if}
+<div class:hidden>
+	{#if category === CategoryEnum.IMAGES}
+		<div class="px-4 py-8">
+			<section id="images" class="flex flex-wrap justify-center gap-2">
+				{#each { length: numberOfResults } as _, i}
+					<div class="flex-none">
+						{#if i % 3 == 0 && i % Math.floor(2 * Math.random()) != 0}
```

Comment:
I did it like this so that the blocks as a whole appeared randomly but evenly coloured. There are better ways to achieve the "evenness" but this way seemed simple enough for it to work aesthetically.